### DISCUSSION TITLE
Issues with Auto-Archive Install. Need assistance

### DISCUSSION THREAD

**Discussion by MikeOne101** [bellingcat/auto-archiver#79] [General]
I need some assistance with installing the auto-archiver. I am receiving an error with the pipenv install. I am getting an error while trying to install the aiohttp package. There might be a problem with the package or my Python environment (no idea).

I am using python 3.11.3

The error reads: Package installation failed.  This is step 2 in the instructional video. 

I have limited knowlege of python so this could be the problem. Any help would be much appreciated.

**Comment by djhmateer** [comment]
I'd try the Docker version perhaps.. (not done so myself yet)..

Otherwise here is my build script for Python 3.9 which I've got working with 3.10 recently
https://github.com/djhmateer/auto-archiver/blob/main/infra/server-build.sh

**Reply by MikeOne101** [reply]
Thank you! I appreciate the response. I will go that route.

**Comment by msramalho** [comment]
Echoing @djhmateer 's answer. If you're not so comfortable around setting up the environment try via docker, that way you don't have to install the associated dependencies on your local machine (ffmpeg, firefox, ...) and only docker. The video does not reflect these new developments which are only documented on the [README.md](https://github.com/bellingcat/auto-archiver#how-to-install-and-run-the-auto-archiver) file.

**Reply by MikeOne101** [reply]
Thanks for the response. I will go that route. I appreciate the additional information.
